lib/matplotlib/legend_handler.py

```diff
@@ -619,6 +624,13 @@ def create_artists(self, legend, orig_handle,
 
         return artists
 
+    def _copy_collection_props(self, legend_handle, orig_handle):
```

Comment:
I managed to fix the legend properties by adding this function, which kind of works.